server/controller/User.js

```diff
@@ -0,0 +1,48 @@
+import Rsvps from '../model/Rsvps';
+import Meetups from '../model/Meetup';
+
+class userController {
+   /**
+    * Create A Rsvp
+    * @param {object} req 
+    * @param {object} res
+    * @returns {object} rsvp object 
+    */
+   static rsvps(req, res) {
```

Comment:
This method name does not reflect what it does, "createRsvps" would be a better choice here

Reply:
done